sass/directives/_carousels.scss

```diff
@@ -249,3 +249,39 @@
     margin-bottom: 0;
   }
 }
+
+@mixin logos-carousel {
+  padding-bottom: $base-padding;
+
+  .slick-track {
+    display: flex;
+    align-items: center;
+
+    img {
+      margin: auto;
+      max-width: 10em;
+      max-height: $homepage-logo-height;
+    }
+  }
+
+  .resource__carousel__dots {
+    display: none !important;
```

Comment:
you could edit https://github.com/cbdr/employer/blob/topic/EM1035-featured-products/app/assets/javascripts/interactions/resource_carousel.js and https://github.com/cbdr/employer/blob/topic/EM1035-featured-products/app/assets/javascripts/interactions/slick_callbacks.js so that `dots: false` can be passed in since that's where the dots are getting added. Then you can avoid the `!important`